front/src/Phaser/Game/GameScene.ts

```diff
@@ -500,10 +497,6 @@ export class GameScene extends DirtyScene implements CenterListener {
             this.outlinedItem?.activate();
         });
 
-        this.presentationModeSprite = new PresentationModeIcon(this, 36, this.game.renderer.height - 2);
-        this.presentationModeSprite.on('pointerup', this.switchLayoutMode.bind(this));
-        this.chatModeSprite = new ChatModeIcon(this, 70, this.game.renderer.height - 2);
-        this.chatModeSprite.on('pointerup', this.switchLayoutMode.bind(this));
```

Comment:
It seems that PresentationModeIcon and ChatModeIcon are not used anymore. You should delete the corresponding classes